### ISSUE TITLE
Update Help for Coords to reflect presence of The Handbook and other needed changes

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/FreeCENMigration#1054]

**Comment by PatReynolds** [comment]
Have made some changes, ready for review.  Unable to link to the Handbook (links needed both in the links near the top, and in the final section).

**Comment by PatReynolds** [comment]
@AlOneill to see if she can figure out what Pat is doing wrong in linking to the Handbook page

**Comment by AlOneill** [comment]
@PatReynolds The DRAFT page is a copy of 'Information for **Transcribers**' rather than for **Coords** (see lead para, for instance). What would you like to do about this?

The Handbook page is not listed when using the 'Insert Link' button (I do not know why not), so the link will need to be inserted manually (in the HTML pane).

**Comment by DeniseColbert** [comment]
@PatReynolds to check

**Comment by PatReynolds** [comment]
@PatReynolds to look at the new version of 'abridged' on the Coords mailing list

**Comment by PatReynolds** [comment]
Version in Test2 (DRAFT Help for Coordinators) now ready for proofing.  Could you take a look @geoffj-FUG for content and @AlOneill for accessibility

**Comment by AlOneill** [comment]
@PatReynolds 
Did you mean test 2 or test 3? I was not able to log in on test 2 but found this page on test 3:
https://test3.freecen.org.uk/cms/information-for-coordinators/draft-information-for-coordinators

The image link shown below points at the FreeCEN home page — is this what was intended? The markup is altogether a little strange, so please have another look.

![FreeCEN-image-link.png](https://images.zenhubusercontent.com/5ce6b34165b1a70acc0a9124/5b162776-a92a-4f7b-b02e-50a8f44b3ffd)

**Comment by AlOneill** [comment]
@PatReynolds 
There is no point in reviewing for accessibility until the content is signed off. My comment above was just something that jumped out at me while trying to find the correct page :-)

**Comment by geoffj-FUG** [comment]
Pat

 

If I looked at this as a new person I would be baffled.

 

How about – ‘We use two reference files in our work, the PARMS file and the Gazetteer.’ Then your text follows.

 

Geoff

**Comment by PatReynolds** [comment]
Have replaced my intro sentence with yours.

**Comment by PatReynolds** [comment]
I don't see any image like tha, Alison.

there is an image missing (called a table here), but I need Geoff's sign off to get it drawn by someone who can draw (and can add that to a later version, I think it helps understanding rather than being essential).

There is a link missing in Enquiries from New Volunteers(  to "Enquiries from potential new volunteers"  that needs to be added (I _think_ this may be the Abbreviated Handbook - so won't be available to potential new volunteers as they won't have access to it.  Do we tell Coords to download and send to potential new volunteers?

**Comment by geoffj-FUG** [comment]
Pat

 

The missing link is I think to the Introduction for new transcribers.

 

I have access to test3 but have never used test2. 

 

I have just tried to sign in but do not have permissions. Denise, can you set me up please?

 

Geoff

**Comment by PatReynolds** [comment]
Hello Geoff,

The page for checking is on test3. Only developers use test2 - it is essentially there to make sure that a new piece of Ruby code plays nicely with the existing code, and functions properly - if it does, the developer moves it to test 3 for wider review.

The html pages only exist on test 3 (and then are copied to live irregularly, when there is something to publish.

With the new version of the handbook, is there any other updating needed?  There are also some areas highlighted by the documentation team which I can address - unless you want to.

If you are signed in to test 3 this page link should work: https://test3.freecen.org.uk/cms/information-for-coordinators/draft-information-for-coordinators

**Comment by AlOneill** [comment]
@PatReynolds 
While tidying up the page, I noted a missing link:

"Enquiries from volunteers will start to come directly to you from FreeCEN2.  A short Introduction booklet LINK HERE is available to assist you to get new Transcribers started."

What is the short introduction referred to here? The Abbreviated Handbook seems unlikely as it could not be described as short! I am stumped.

**Comment by AlOneill** [comment]
@PatReynolds page tidied and some typos fixed. Only the missing link to resolve.

**Comment by AlOneill** [comment]
@PatReynolds DRAFT page promoted and sync-ready.

**Comment by PatReynolds** [comment]
@Vino-S please sync

**Comment by geoffj-FUG** [comment]
Pat

 

(Denise, Found it on test3 so don’t need access to test2. Geoff)

Typo as follows: be ualso pdated

I am wondering why some of this is going online. I know it was there on FC1 but we now have the Handbooks. FC1 had very rudimentary guidelines.

I am updating the handbooks. The Abbreviated Handbook will become Transcribers and Proofreaders Handbook next issue. The Full Handbook will become that plus two or three supplements. 

The Validators Supplement will be issued in about a week. I am just waiting on Story #1384 as it is written for after that story is completed.

Then there will be a Coordinator’s supplement.

It means that the Supplements will not contain the info in the Transcribers Handbook such as the field descriptions.

The nett result is that there will only ever be one version of anything published. At the moment we have redundancy as there is a Transcribers version and a Full version which contain the same info. If they get out of synch there will be confusion about instructions. So I am going for a do it only once model.

Some of the Info in this text is a duplicate of what is in the Handbook. Are we building in redundancy?

Geoff
